userspace/libpman/src/sc_set.c

```diff
@@ -69,7 +93,54 @@ int pman_enforce_sc_set(bool *sc_set) {
 		sched_prog_exec = true;
 	}
 
-	/* Enable desired tracepoints */
+	if(sys_exit) {
```

Comment:
No need to attach TOCTOU mitigation programs (dispatchers) if the `sys_exit` dispatcher is not attached. The reason behind this is that enter events, generated by the tail-called TOCTOU mitigation programs, are conceived to support exit events.